# Pruebas de bondad de ajuste

**Prueba Kolmogorov**

In [5]:
datos<-c(0.14, 0.16, 0.28, 0.37, 0.42, 0.55, 0.83, 0.87, 0.92, 0.99)

In [2]:
ks.test(datos, "punif")


	One-sample Kolmogorov-Smirnov test

data:  datos
D = 0.23, p-value = 0.5886
alternative hypothesis: two-sided


Como $\textrm{P}_{valor}=P_(H_0)(D_n>0.23)=0.5886$
$\Rightarrow$ No se rechaza la $H_0$